README.md

```
@@ -3,14 +3,136 @@
+npm ci
 ```
-npm install
+
+`npm ci` rather than `npm install`, so you get exactly what `package-lock.json` pins.
+
+There is an `.npmrc` setting `legacy-peer-deps=true`, and it is load-bearing. `eslint`
+is pinned at `^5.16.0` while `eslint-config-standard@^13.0.1` declares a peer of
+`eslint >= 6.0.1`. npm 6, which this project was written against, ignored peer
+conflicts. npm 7 and later refuse to install at all:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)